cli/cli.hs

```diff
@@ -574,19 +607,20 @@ symvmFromCommand cmd calldata = do
     callvalue = maybe TxValue Lit cmd.value
     storageBase = maybe AbstractBase parseInitialStorage (cmd.initialStorage)
 
-  contract <- case (cmd.rpc, cmd.address, cmd.code) of
+  codeWrapped <- getCode "--code --code-file" cmd.codeFile cmd.code
```

Comment:
But here you did not include "and" in the message. Shouldn't the message be the same?

Actually, wouldn't it be better to just hardcode the error message in `getCode`? Do you really need to customize it?
Can you just drop the `msg` parameter of `getCode`?

Reply:
Yeah, good point. I removed the message.

I will also be creating a new PR for common options, as you noted in your other review, we definitely do need it. I'll see how it can be done.